transactions/add-account/2025/jan-21/add_fvm_metering_account.cdc

```diff
@@ -0,0 +1,57 @@
+import FlowToken from 0x1654653399040a61
+import FungibleToken from 0xf233dcee88fe0abe
+import FlowServiceAccount from 0xe467b9dd11fa00df
+
+// 1. Creates a new account
+// 2. Creates an account capability for the new account and stores it in the service account
+// 3. Transfers 5 FLOW to the new account from the signer account
+// 4. Moves metering settings over to this new account
+
+transaction(name: String, code: String) {
+    let tokenReceiver: &{FungibleToken.Receiver}
+    let sentVault: @{FungibleToken.Vault}
+
+    prepare(signer: auth(AddKey, BorrowValue) &Account) {
```

Comment:
I tested this on seperate account on testnet and got this error - 
```
error: cannot access `save`: function requires `Storage | SaveValue` authorization, but reference only has `AddKey, BorrowValue` authorization
  --> a103b6431a23cbe6b17373dd0be00723ea8abf3f48536617d1b4fac6e5178310:24:8
   |
24 |         signer.storage.save(capability, to: /storage/meteringSettingsAccount)
   |         ^^^^^^^^^^^^^^^^^^^
```
fix would be
```suggestion
    prepare(signer: auth(Storage, SaveValue, BorrowValue) &Account) {
```